examples/jsm/utils/BufferGeometryUtils.d.ts

```diff
@@ -1,7 +1,7 @@
 import { BufferAttribute, BufferGeometry } from '../../../src/Three';
 
 export namespace BufferGeometryUtils {
-    export function mergeBufferGeometries(geometries: BufferGeometry[]): BufferGeometry;
+    export function mergeBufferGeometries(geometries: BufferGeometry[], useGroups: boolean): BufferGeometry;
```

Comment:
Please mark the parameter as optional like so: `useGroups?`

Reply:
whoops. that was lost in translation. fixed.